docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,137 @@
+ATC should declare the circuit saturated when aircraft exceeds the below table.
+When the circuit is saturated further circuits will be unavailable until numbers are within tolerance of the table.
+This number may be reduced by ATC due to complexity of operations and prevailing weather in order to ensure safety.
+
+| Time   | ESL Day | ESL Night | WSL Day | WSL Night |
+| ------ | ------- | --------- | ------- | --------- |
+| Day    | 6       | 5         | 5       | 5         |
+
+## Low Approach
+- By day, pilots of local aircraft may request a low approach.
+- Pilots are responsible for ensuring that no collision risk exists and that there is suitable spacing to continue the approach and for the potential/subsequent go-around.
+
+## Standard Taxi Routes
+| Runway     | Outbound Route                   | Inbound Route                   |
+| ---------- | -------------------------------- | -------------------------------- |
+| Runway 27  | E2 or Eastern Check Bays         | D2                               |
+| Runway 09  | D2 or Western Check Bays         | E2                               |
+| Runway 22  | E2 or Eastern Check Bays         | J                                |
+| Runway 04  | J                                | D2                               |
+
+## Helicopter Operations
+- **Helicopter Circuits**:  
+  - When RWY09/27 is being used for fixed-wing circuit training, it is preferred that helicopters utilise threshold runway 04 (runway 09/27 direction) for landing and take-off to increase segregation between final approach and upwind segments.
+  - When helicopters require use of TWY A (Pad Alpha) for circuit training, base turns should be sequenced to avoid conflict during the final approach segment.
+
+## Lanes
+- **Lanes Overview**:  
+  The 16 individual training areas within the ESL military airspace are separated by 4 outbound lanes.
+  These lanes are used aircraft transiting to/from exterior training areas or for entry and exit of ESL military airspace.
+  Lanes are defined by GNSS waypoints situated at 12, 35, and 50 NM from YMES AD.
+  
+- **Outbound Lanes**:
+
+| Lane Direction | 12NM GNSS Waypoint | 35NM GNSS Waypoint | 50NM GNSS Waypoint | Bearing |
+| -------------- | ------------------ | ------------------ | ------------------ | ------- |
+| Northern       | LEKEM              | GONEB              | LANOS              | 359     |
+| Eastern        | VEMDA              | KADRU              | TAVET              | 090     |
+| Southern       | SABAX              | LUTUK              | NOLOX              | 180     |
+| Western        | DUGAD              | LERKO              | DUNNE              | 270     |
+
+- **Vertical Dimensions**:  
+  Lanes extend from SFC or the base of restricted airspace to `F160`.
+
+- **Departure Instructions**:  
+  For departure instructions with a heading assignment, ATC must ensure that an assigned heading or a climb requirement is provided to segregate or separate with STA activity.
```

Comment:
What is STA out of curiosity? Could you maybe look at defining it in the abbreviations page?

Reply:
Specified Training Areas (STA)

Little bit of a copy/paste that one, but thats what it is in the ES mil AIP.

Can remove it since if you guys are getting confused then normal controllers will